core/build.gradle

```diff
@@ -18,5 +18,9 @@ android {
 
 dependencies {
     implementation project(':database')
+
```

Comment:
not necessary in core anymore

Reply:
actually still needed (at least in my version), some classes use it...